### PR TITLE
Support multiple webhook targets

### PR DESCRIPTION
**Pull request by msterle**
# Pull Request

## Checklist
- [x] Have you read Digital Catapult's [Code of Conduct](https://github.com/digicatapult/.github/blob/main/CODE_OF_CONDUCT.md)?
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.

## PR Type

Please delete options that are irrelevant.

- [ ] Bug Fix
- [x] Feature
- [ ] Documentation Update
- [ ] Code style update (formatting, local variables)
- [ ] Breaking Change (fix or feature that would cause existing functionality to change)

## Linked tickets

n/a

## High level description

Support multiple webhook targets by converting `webhook-url` config option from a string to an array of strings.

## Detailed description

This is especially needed for NICE deployment where events need to propagate to both the Query Agent and the Peer API

## Describe alternatives you've considered

- renaming config option to pluralised `webhookUrls`, but this would either break existing configs, or require a transition and deprecation period.

## Operational impact

none

## Additional context

n/a

### CODE REVIEW

File: src/events/BasicMessageEvents.ts
Comment (Ellenn-A) [inline src/events/BasicMessageEvents.ts:15]: so if there are multiple webhooks we no longer await the send webhook events?

Comment (msterle) [reply src/events/BasicMessageEvents.ts:15]: we never should have awaited it, since we don't want or need to wait for the response.

File: src/cli.ts
Comment (dblane-digicatapult) [inline src/cli.ts:100]: Can you please update the helm chart once this has been merged.

**Review by Ellenn-A — DISMISSED** [review]

**Review by jonmattgray — APPROVED** [review]

**Pull request merged**
